studio/data/fetchers.ts

```diff
@@ -89,9 +89,11 @@ export const del: typeof _del = async (url, init) => {
 export const head: typeof _head = async (url, init) => {
   const headers = await constructHeaders(init?.headers)
 
+  // @ts-ignore
```

Comment:
Why is this needed here?

Reply:
ahh i'm only running into a ts error when i added the parseAs: 'text' - wondering if you might know whats the cause as i'm not really sure :/ 
<img width="816" alt="image" src="https://github.com/supabase/supabase/assets/19742402/6f748fb8-a246-41cd-a1a7-4b6df92a1336">

Reply:
I think it's because it doesn't know the endpoint, and so it doesn't know if it supports `parseAs` yet. If you pass in `parseAt` at the call site instead of in this file, it shouldn't cause a type error.